src/CodeEditor/CodeEditor.css

```diff
@@ -1,14 +1,17 @@
 .ace_editor {
```

Comment:
the CSS for this module does not work? i couldnt figure out how to get it to work if anyone has inputs here

Reply:
o i think it has to be renamed to be `CodeEditor.module.css` (and the import should be updated in the index.tsx file too) - lmk if that works!

Reply:
got the css working if the import in index.tsx is changed from `import styles from "./CodeEditor.css";` to `import "./CodeEditor.css";`

Reply:
this worked ty!